src/server_manager/web_app/app.ts

```diff
@@ -38,6 +38,9 @@ interface PolymerEvent extends Event {
 const UNUSED_DIGITALOCEAN_REFERRAL_CODE = '5ddb4219b716';
 
 const CHANGE_KEYS_PORT_VERSION = "1.0.0";
+const DATA_LIMITS_VERSION = '1.1.0';
```

Comment:
Do you want to include both this and the hostname stuff together on the next Shadowbox release?

Reply:
Sure!